pkg/sources/github/github.go

```diff
@@ -919,3 +925,141 @@ func (s *Source) setProgressCompleteWithRepo(index int, offset int, repoURL stri
 
 	s.SetProgressComplete(index+offset, len(s.repos)+offset, fmt.Sprintf("Repo: %s", repoURL), encodedResumeInfo)
 }
+
+func (s *Source) scanComments(ctx context.Context, repoPath string, chunksChan chan *sources.Chunk) error {
+	s.log.Info("scanning comments")
+
+	trimmedURL := removeURLAndSplit(repoPath)
+	owner := trimmedURL[1]
+	repo := trimmedURL[2]
+
+	sortType := "created"
```

Comment:
nit: Grouping related vars tends to read a bit nicer imo.
ex:
```
	var (
		sortType      = "created"
		directionType = "desc"
	)
```

Reply:
ahhh true